tests/unit/modules/test_chroot.py

```diff
@@ -42,7 +43,17 @@ class ChrootTestCase(TestCase, LoaderModuleMockMixin):
     """
 
     def setup_loader_modules(self):
-        return {chroot: {"__salt__": {}, "__utils__": {}, "__opts__": {"cachedir": ""}}}
+        loader_context = salt.loader.context.LoaderContext()
```

Comment:
Both of these tests do not fail without this fix. Can you add a regression test that will catch this in the future?